packages/flutter/test/widgets/listener_deprecated_test.dart

```diff
@@ -178,7 +178,7 @@ void main() {
         ),
       ));
       expect(exit, isNull);
-      expect(tester.binding.mouseTracker.isAnnotationAttached(renderListener.hoverAnnotation), isFalse);
+      expect(tester.binding.mouseTracker.isAnnotationAttached(renderListener.annotationKey), isFalse);
```

Comment:
I believe the new breaking change policy would consider these changes as breaking.

Reply:
The `isAnnotationAttached` is a `visibleForTest` method, and is not used in g3. If this is a concern I can use a 2-phase deprecation instead.